db/migrate/20210118102533_add_cip_content.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class AddCipContent < ActiveRecord::Migration[6.1]
+  def change
+    create_table :course_years, id: :uuid do |t|
+      t.timestamps
+      t.column :is_year_one, :boolean, null: false
+      t.column :title, :string, null: false
+      t.column :content, :string, null: false
```

Comment:
Not sure what content looks like but if it's anything the size of  a document then a text field might be more appropriate.

Reply:
Good call!